flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -54,6 +54,9 @@ case class Project(projectList: Seq[NamedExpression], child: LogicalNode) extend
             case expr if !expr.valid => u
             case c @ Cast(ne: NamedExpression, tp) => Alias(c, s"${ne.name}-$tp")
             case gcf: GetCompositeField => Alias(gcf, gcf.aliasName().getOrElse(s"_c$i"))
+            case over: OverCall if null != over.aggAlias =>
```

Comment:
If i understand you correctly, you want add alias for AGG,e.g.:
` .select('c, 'b.count over 'w as 'mycount) -->  .select('c, 'b.count  as 'mycount over 'w)`
 but for SQL we add alias to OVER, e.g.:
`SELECT c, count(a) OVER (PARTITION BY c ORDER BY ProcTime()  RANGE UNBOUNDED preceding) as cnt1 from T1`
So  I want to be consistent with SQL，What do you think?